config/filament.php

```diff
@@ -10,9 +10,14 @@
     | The default is `admin` but you can change it to whatever works best and
     | doesn't conflict with the routing in your application.
     |
+    | You may also change the domain where Filament should be active. If the
+    | domain is empty, all domains will be valid.
+    |
     */
 
     'path' => 'admin',
+    
+    'domain' => '',
```

Comment:
I'd add a comment above the config option, like the `path` one above.

Reply:
Also, I think it might make sense to allow using an `env()` here too, that way you could change the domain based on staging, production, local, etc.

Reply:
@ryangjchandler yes I agree, I updated my PR